pkg/sdn/plugin/node.go

```diff
@@ -124,7 +133,24 @@ func (node *OsdnNode) Start() error {
 }
 
 func (node *OsdnNode) GetLocalPods(namespace string) ([]kapi.Pod, error) {
-	return node.registry.GetRunningPods(node.hostName, namespace)
+	fieldSelector := fields.Set{"spec.host": node.hostName}.AsSelector()
```

Comment:
hm... actually, it's referring to a deprecated v1.PodSpec field which was replaced with "nodeName", not "hostname", which is apparently something slightly different.